_documentation/automation.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Automation
+title: documentation.categories.automation.title
 layout: doc
 level: "1"
 group: "automation"
 icon: "fas fa-robot"
 order: "60"
 ---
 
-# Automation
+# {% t {{ page.title }} %}
 
-Are you an experienced user and want to automate tasks? Have a look at the pages below.
+{% t documentation.categories.{{ page.group }}.intro %}
```

Comment:
I would actually move this to markdown as well. Then we could add more detailed explanations when needed.

Reply:
I think the introductions should be short, and without any markup. I want to style them a bit further (via CSS). Including them in the markdown files makes it more likely that the introductions will become messy.